src/Utils/AuthorizationUtils.ts

```diff
@@ -1,21 +1,27 @@
 import * as msal from "@azure/msal-browser";
+import { EncryptedAccessToken } from "Contracts/DataModels";
 import { AuthType } from "../AuthType";
 import * as Constants from "../Common/Constants";
+import { HttpHeaders } from "../Common/Constants";
 import * as Logger from "../Common/Logger";
 import { configContext } from "../ConfigContext";
-import * as ViewModels from "../Contracts/ViewModels";
 import { userContext } from "../UserContext";
 
-export function getAuthorizationHeader(): ViewModels.AuthorizationTokenHeaderMetadata {
+export function getAuthorizationHeaders() {
   if (userContext.authType === AuthType.EncryptedToken) {
-    return {
-      header: Constants.HttpHeaders.guestAccessToken,
-      token: userContext.accessToken,
-    };
+    return userContext.accessToken ? getAccessTokenAuthorizationHeaders(userContext.accessToken) : {};
+  } else {
+    return { [Constants.HttpHeaders.authorization]: userContext.authorizationToken || "" };
```

Comment:
This will change the returned object to:
`
{
 	authorization: "token value"
}
`

The returned object before was:
`
{
	header: authorization,
	token: "token value"
}
`

Is this intentional?

Reply:
Yes, this is intentional. I have changed the getAuthorizationHeaders return type to be more easily consumed by it's various callers.